cmd/nomos/util/configmanagement.go

```diff
@@ -161,3 +170,30 @@ func (c *ConfigManagementClient) IsMultiRepo(ctx context.Context) (*bool, error)
 	}
 	return &isMulti, nil
 }
+
+// IsOssInstallation will check for the existence of ConfigManagement object, Operator deployment, and RootSync CRD
+// If RootSync CRD exist but ConfigManagement and Operator doesn't, it indicates an OSS installation
+func IsOssInstallation(ctx context.Context, c *ConfigManagementClient, cl client.Client, ck *kubernetes.Clientset) (bool, error) {
+	v, cmErr := c.Version(ctx)
+	if cmErr != nil {
+		err := fmt.Errorf("Failed to get the ConfigManagment version: %v", cmErr)
+
+		return false, err
```

Comment:
nit: You can`return false, fmt.Errorf("Failed to get the ConfigManagment version: %v", cmErr)` directly without declaring `err`. 
Same for the following blocks.